### ISSUE TITLE
Please bring back the user count on /about.

### ISSUE THREAD

**Issue by egypturnash** [mastodon/mastodon#26262]
### Pitch

Being able to see how many people are on a particular instance is super important when making admin decisions. Why was this removed?

### Motivation

Me. I would benefit from it. An important part of making admin decisions is "how big is this instance I'm looking at". And an important part of trying to offer suggestions for other admin's decisions is "how big is the instance you're running on". I have been finding myself constantly annoyed at this change.

**Comment by Gargron** [comment]
The user count is irrelevant if those people are inactive. The active user number provides a more accurate representation of server size.

**Comment by egypturnash** [comment]
You are correct, the active user number is actually the one I am looking for.

It's gone too. Why was it removed?

**Comment by Gargron** [comment]
It's on the left:

![image](https://github.com/mastodon/mastodon/assets/184731/ca250d07-6653-4ba1-9c9c-1cc36bc609f3)

**Comment by egypturnash** [comment]
Ah, it is indeed! But only on desktop. It’s completely hidden on my phone and my tablet.

Can we get it out back in the area that doesn’t collapse in mobile views? Thanks!
